bentoml/configuration/containers.py

```diff
@@ -277,3 +278,6 @@ class BentoMLContainer(containers.DeclarativeContainer):
         ),
         config.yatai.logging.path,
     )
+
+
+BentoMLContainer = BentoMLContainerClass()
```

Comment:
is this naming convention coming from the other dependency_injector library?
any reason not to just do `default_container = BentoMLContainer()`

Reply:
discussed offline, we will make the following rename in a follow up PR to reduce the changes here:

```
BentoMLContainerClass => BentoMLContainer
BentoMLContainer => BENTOML_CONTAINER
```